sidebars.js

```diff
@@ -43,8 +43,7 @@ module.exports = {
               items: [
                 'guides/setup/configuring_data_contexts/how_to_configure_a_new_data_context_with_the_cli',
                 'guides/setup/configuring_data_contexts/how_to_configure_datacontext_components_using_test_yaml_config',
-                'guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_yaml_file_or_environment_variables',
-                'guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_secrets_store',
+                'guides/setup/configuring_data_contexts/how_to_configure_credentials',
```

Comment:
For any files we delete, you need to include an entry in ../static/_redirects; otherwise approved!

Reply:
Please review my entry in `_redirects`. Thanks!